kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js

```diff
@@ -774,13 +773,9 @@ define([
 
                 let shareCount = -1; // our user is always going to be included, but will bump the count, since it's not "shared"
                 const perms = this.wsPerms[data.ws[0]];
-                for (const usr in perms) {
-                    if (perms.hasOwnProperty(usr)) {
-                        if (usr === '*') {
-                            continue;
-                        }
-                        shareCount++;
-                    }
+                shareCount += Object.keys(perms).length;
+                if (perms['*']) {
+                    shareCount--;
```

Comment:
The `hasOwnProperty` bit threw a linting error. I couldn't resist a chance to get rid of a loop anyway.